pandas-stubs/core/series.pyi

```diff
@@ -72,6 +72,26 @@ class Series(Generic[_DType]):
     def all(self, axis: int = ..., bool_only: bool = ...) -> bool: ...
     def any(self, axis: int = ..., bool_only: bool = ...) -> bool: ...
     @overload
+    def append(
+        self, to_append: Series, ignore_index: bool = ..., verify_integrity: bool = ...
+    ) -> Series: ...
+    @overload
+    def append(
+        self, to_append: List[Series], ignore_index: bool = ..., verify_integrity: bool = ...
+    ) -> Series: ...
+    @overload
+    def append(
+        self, to_append: Tuple[Series], ignore_index: bool = ..., verify_integrity: bool = ...
+    ) -> Series: ...
+    @overload
+    def apply(
+        self, func: Callable, convert_dtype: bool = ..., args: Tuple = ..., **kwargs: Any
+    ) -> Series: ...
+    @overload
+    def apply(
+        self, func: Callable, convert_dtype: bool = ..., args: Tuple = ..., **kwargs: Any
+    ) -> DataFrame: ...
```

Comment:
The problem is here probably. you have two definitions of `apply()` that are completely identical, but they return different types. That's not allowed for overloading (because then you cannot know what the return type is).